packages/css/src/table.css

```diff
@@ -2,27 +2,26 @@
   --dsc-table-background--hover: var(--ds-color-neutral-surface-default);
   --dsc-table-background--zebra: var(--ds-color-neutral-background-subtle);
   --dsc-table-background: var(--ds-color-neutral-background-default);
-  --dsc-table-border-radius: 0;
   --dsc-table-border-color: var(--ds-color-neutral-border-subtle);
+  --dsc-table-border-divider: 2px solid var(--dsc-table-border-color);
+  --dsc-table-border-radius: var(--ds-border-radius-sm);
+  --dsc-table-border: 1px solid var(--dsc-table-border-color);
```

Comment:
This breaks the convention set in #3002 - see last bulletpoint.
`border:` should have one custom property, width and style.
`border-color:` should be set outside of this, and on it's on property.

This is to ensure it is easier to update either border width and style, without needing to know the var for color

Reply:
But, can't the consumer use `border-width` individually stil? ☺️ 
Was unsure about this one to, but made a composed `border` variable since we use it multiple times, and it is hard to know wether it is border-bottom or border-top used between rows 🤷 
But, maybe we should instead have a `--dsc-table-border-between` to clarify where it is used? 🤔 we'll probably need some minor adjusments anyway ☺️